gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.6.1-bin.zip
```

Comment:
Oh, there's also a gradle update and Kotlin version update hidden here. Yeah, usually preferred this in separate commits. :)

Reply:
Sorry about that, i reverted it.